dto/src/main/java/ca/on/oicr/gsi/runscanner/dto/OxfordNanoporeNotificationDto.java

```diff
@@ -3,8 +3,18 @@
 import ca.on.oicr.gsi.runscanner.dto.type.Platform;
 
 public class OxfordNanoporeNotificationDto extends NotificationDto {
+  private String scriptPurpose;
```

Comment:
What do you think about the term `runType` for this? (I don't know that `scriptPurpose` necessarily gives a huamn reading the DTO as much context for what the field means)

Reply:
Sounds good!